c/zss.c

```diff
@@ -731,18 +767,25 @@ static void printZISStatus(HttpServer *server) {
 
 }
 
-static void readAgentAddressAndPort(JsonObject *serverConfig, char **address, int *port) {
+static void readAgentAddressAndPort(JsonObject *serverConfig, JsonObject *envConfig, char **address, int *port) {
+  *port = jsonObjectGetNumber(envConfig, "ZWED_agent_http_port");
```

Comment:
This does not appear to be the same JSON as what app server builds. It would be simpler to just avoid the JSON idea entirely rather than have it incompatible:

This:
```
{
"ZWED_agent_http_port": 12345,
"ZWED_agent_http_ipAddresses": "1.2.3.4",
}
```

app server:

```
{
  "agent": {
    "http": {
      "port": 12345,
      "ipAddresses": "1.2.3.4"
    }
  }
}
```

The reason to have JSON at all was to be able to merge, so if not merging, no need for JSON.